### DISCUSSION TITLE
Meetings with same adress but different locations at same address

### DISCUSSION THREAD

**Discussion by mikepoconnor4** [code4recovery/12-step-meeting-list#844] [Q&A]
I'm having a strange issue. I have a large location (the Kahala Mall in Honolulu - 4211 Waialae Ave, Honolulu, HI) and there's 2 different suite numbers there with different meetings. When I try to add the Suite number, the geocoding strips it back down to the base address - 4211 Waialae Ave. After that I can't distinguish the 2 locations with 2 different church names and location notes with separate directions. Is there a workaround for this?

**Comment by tech2serve** [comment]
Hi Mike,

The simplest solution is to include the suite number in the notes, but I realize that may less than ideal.

Another option might be to see what you can accomplish with [address overrides](https://github.com/code4recovery/12-step-meeting-list/wiki/Frequently-Asked-Questions#i-need-to-correct-a-meeting-address-or-change-a-pins-location). My thought would be that if you designate each by including their suite number, as long as you add that exactly when you enter the address, the override might put the pin in the correct location. I believe you'll still need to put the church name in the location notes.

I've not experimented with this so I'm really interesting in hearing if this works.

**Reply by alangrady** [reply]
This problem came up for me as well.

Same building, but they different location notes.

" Follow sidewalk to entrance on left side of building (Horne Street side) and go down stairs." 

And

"Go to top floor of fire escape stairs, first room on left"

Changed one of them, both meetings change. 😱

**Reply by Grateful-Viv** [reply]
It was suggested to me a while back when this happened to put the information in the meeting note area rather than location note area. That way each meeting can have its own unique note.

**Reply by alangrady** [reply]
Yes... I did that.  But, it Should be next to the Location. 😄
